src/coord/src/coord.rs

```diff
@@ -2383,7 +2386,7 @@ impl Coordinator {
         // the compacted arrangements we have at hand. It remains unresolved
         // what to do if it cannot be satisfied (perhaps the query should use
         // a larger timestamp and block, perhaps the user should intervene).
-        let (index_ids, indexes_complete) = self.catalog.nearest_indexes(uses_ids);
+        let (index_ids, source_ids) = self.catalog.nearest_indexes(uses_ids);
```

Comment:
`source_ids` => `unmaterialized_ids` or something I think would be clearer. `source_ids` implies the opposite of what it is.

Reply:
done